From bd41d89c0f603c0108e110343d9fbbaa3bfa0cc8 Mon Sep 17 00:00:00 2001
From: Shivam Mathur <shivam_jpr@hotmail.com>
Date: Mon, 15 Dec 2025 22:19:40 +0530
Subject: [PATCH] Curl patch

---
 CMakeLists.txt | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c0cc0c57b..2d499c059 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -168,13 +168,19 @@ if(CMAKE_CROSSCOMPILING)
     "${CMAKE_SYSTEM_NAME}/${CMAKE_SYSTEM_PROCESSOR}")
 endif()
 
-if(CMAKE_C_COMPILER_TARGET)
+if(WIN32)
+  if(CMAKE_SIZEOF_VOID_P EQUAL 8)
+    set(CURL_OS "\"x86_64-pc-win32\"")
+  else()
+    set(CURL_OS "\"i386-pc-win32\"")
+  endif()
+elseif(CMAKE_C_COMPILER_TARGET)
   set(CURL_OS "\"${CMAKE_C_COMPILER_TARGET}\"")
 else()
   set(CURL_OS "\"${CMAKE_SYSTEM_NAME}\"")
 endif()
 
-set(LIB_NAME "libcurl")
+set(LIB_NAME "libcurl_a")
 
 set_property(DIRECTORY APPEND PROPERTY INCLUDE_DIRECTORIES "${PROJECT_SOURCE_DIR}/include")
 
-- 
2.50.1 (Apple Git-155)

