diff --git a/src/CommandLineInfo.cpp b/src/CommandLineInfo.cpp
index a20cbe1..0f9ab5b 100644
--- a/src/CommandLineInfo.cpp
+++ b/src/CommandLineInfo.cpp
@@ -56,6 +56,8 @@ void CommandLineInfo::ParseParam(const wchar_t* pszParam, BOOL bFlag, BOOL bLast
     _options->useHDRI=FALSE;
   else if (_wcsicmp(pszParam, L"noOpenMP") == 0)
     _options->useOpenMP=FALSE;
+  else if (_wcsicmp(pszParam, L"noOpenCL") == 0)
+    _options->useOpenCL=FALSE;
   else if (_wcsicmp(pszParam, L"noWizard") == 0)
     showWizard=false;
   else if (_wcsicmp(pszParam, L"limitedPolicy") == 0)
diff --git a/src/Options.cpp b/src/Options.cpp
index e1019eb..5c96ccb 100644
--- a/src/Options.cpp
+++ b/src/Options.cpp
@@ -65,10 +65,7 @@ const wstring Options::channelMaskDepth() const
   if (!isImageMagick7)
     return(L"");
 
-  if ((visualStudioVersion >= VisualStudioVersion::VS2022) && (architecture != Architecture::x86))
-    return(L"64");
-  else
-    return(L"32");
+  return(L"32");
 }
 
 const wstring Options::platform() const
diff --git a/src/Project.cpp b/src/Project.cpp
index 60137fb..717622c 100644
--- a/src/Project.cpp
+++ b/src/Project.cpp
@@ -33,7 +33,7 @@ const wstring Project::characterSet() const
 
 Compiler Project::compiler() const
 {
-  return(_config.isMagickProject() && _options.visualStudioVersion >= VisualStudioVersion::VS2022
+  return(_config.isMagickProject() && true
     ? Compiler::CPP
     : Compiler::Default);
 }
