diff --git a/MagickCore/memory.c b/MagickCore/memory.c
index 41e84be..7e33f52 100644
--- a/MagickCore/memory.c
+++ b/MagickCore/memory.c
@@ -202,12 +202,35 @@ static size_t
   max_profile_size = 0,
   virtual_anonymous_memory = 0;
 
+#if defined _MSC_VER
+static void *MSCMalloc(size_t size)
+{
+  return(malloc(size));
+}
+
+static void *MSCRealloc(void* ptr, size_t size)
+{
+  return(realloc(ptr,size));
+}
+
+static void MSCFree(void* ptr)
+{
+  free(ptr);
+}
+#endif
+
 static MagickMemoryMethods
   memory_methods =
   {
+#if defined _MSC_VER
+    (AcquireMemoryHandler) MSCMalloc,
+    (ResizeMemoryHandler) MSCRealloc,
+    (DestroyMemoryHandler) MSCFree,
+#else
     (AcquireMemoryHandler) malloc,
     (ResizeMemoryHandler) realloc,
     (DestroyMemoryHandler) free,
+#endif
     (AcquireAlignedMemoryHandler) NULL,
     (RelinquishAlignedMemoryHandler) NULL
   };
diff --git a/MagickCore/vms.c b/MagickCore/vms.c
index dae88e7..96248c4 100644
--- a/MagickCore/vms.c
+++ b/MagickCore/vms.c
@@ -269,3 +269,8 @@ MagickExport MagickBooleanType VMSIsMagickConflict(const char *magick)
   return(MagickFalse);
 }
 #endif /* defined(vms) */
+
+/* Ensure non-empty translation unit for MSVC */
+#if defined(_MSC_VER) && !defined(vms)
+typedef int MagickVMSDummy;
+#endif
diff --git a/coders/png.c b/coders/png.c
index 42a854a..b6ac136 100644
--- a/coders/png.c
+++ b/coders/png.c
@@ -42,6 +42,9 @@
 */
 #include "MagickCore/studio.h"
 #include "MagickCore/artifact.h"
+#if defined(_MSC_VER)
+#pragma warning(disable: 4197)
+#endif
 #include "MagickCore/attribute.h"
 #include "MagickCore/blob.h"
 #include "MagickCore/blob-private.h"
