 Makefile.am         | 4 ----
 test/alg-yescrypt.c | 2 +-
 2 files changed, 1 insertion(+), 5 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 548d44a..7e7dafb 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -378,12 +378,9 @@ check_PROGRAMS = \
 	test/alg-sm3 \
 	test/alg-sm3-hmac \
 	test/alg-yescrypt \
-	test/badsalt \
 	test/badsetting \
 	test/byteorder \
 	test/checksalt \
-	test/compile-strong-alias \
-	test/crypt-badargs \
 	test/crypt-gost-yescrypt \
 	test/crypt-nested-call \
 	test/crypt-sm3-yescrypt \
@@ -395,7 +392,6 @@ check_PROGRAMS = \
 	test/gensalt-nested-call \
 	test/gensalt-nthash \
 	test/getrandom-fallbacks \
-	test/getrandom-interface \
 	test/preferred-method \
 	test/short-outbuf \
 	test/special-char-salt
diff --git a/test/alg-yescrypt.c b/test/alg-yescrypt.c
index baa141d..a24a4a9 100644
--- a/test/alg-yescrypt.c
+++ b/test/alg-yescrypt.c
@@ -159,7 +159,7 @@ int main(void)
 {
 	int i;
 
-	setvbuf(stdout, NULL, _IOLBF, 0);
+	// setvbuf(stdout, NULL, _IOLBF, 0);
 
 #ifdef TEST_PBKDF2_SHA256
 	print_PBKDF2_SHA256("password", "salt", 1, 20);
