 build/cmake/lib/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/build/cmake/lib/CMakeLists.txt b/build/cmake/lib/CMakeLists.txt
index 4e902a1f..d8860c2c 100644
--- a/build/cmake/lib/CMakeLists.txt
+++ b/build/cmake/lib/CMakeLists.txt
@@ -204,7 +204,7 @@ if (ZSTD_BUILD_SHARED)
     set_target_properties(
             libzstd_shared
             PROPERTIES
-            OUTPUT_NAME zstd
+            OUTPUT_NAME libzstd
             VERSION ${ZSTD_FULL_VERSION}
             SOVERSION ${zstd_VERSION_MAJOR})
             
@@ -234,7 +234,7 @@ if (ZSTD_BUILD_STATIC)
             libzstd_static
             PROPERTIES
             POSITION_INDEPENDENT_CODE On
-            OUTPUT_NAME ${STATIC_LIBRARY_BASE_NAME})
+            OUTPUT_NAME libzstd_a)
 
     if (ZSTD_FRAMEWORK)
         set_target_properties(

